src/main/java/mediathek/config/Daten.java

```diff
@@ -326,8 +326,8 @@ public void launchHistoryDataLoading() {
     }
 
     private ListenableFuture<SeenHistoryController> launchSeenHistoryController(ListeningExecutorService pool) {
-        var historyFuture = pool.submit(new SeenHistoryCallable());
```

Comment:
apart from your personal preferences what is the benefit of this change?

Reply:
A filed with the name "historyFuture" is already defined in line 102, redefining it in the function with the same name triggers warnings about hidden fields.